firefox-ios/Shared/Strings.swift

```diff
@@ -7574,6 +7574,22 @@ extension String {
     }
 }
 
+// MARK: - Live Activity
+extension String {
+    public struct LiveActivity {
+        public static let DownloadingFile = MZLocalizedString(
+            key: "LiveActivity.DownloadingFile.v135",
+            tableName: "LiveActivity",
+            value: "Downloading %@",
+            comment: "Displayed during a download in Live Activity or Dynamic Island. The %@ placeholder represents either the filename of the file being downloaded surrounded by double quotes (e.g. Downloading \"MyFile.pdf\") or a count of multiple files (e.g. Downloading 2 files).")
+        public static let DownloadingProgress = MZLocalizedString(
+            key: "LiveActivity.DownloadingProgress.v135",
```

Comment:
Sames as one of my last comments, I believe the version that we should be targeting here is v137 instead of v135.